docs/guides/admin/docs/configuration/admin-ui/terms.md

```diff
@@ -14,6 +14,6 @@ The content of the terms can be set in `/etc/ui-config/mh_default_org/admin-ui/`
 terms.%{language_code}.html
 ```
 
-The corresponding language codes can be found here: https://github.com/opencast/opencast-admin-interface/blob/admin-ui-picard/app/src/i18n/i18n.ts#L44
+The corresponding language codes can be found in [`/src/i18n/i18n.ts`](https://github.com/opencast/opencast-admin-interface/blob/4253105caedb11df193c05188e043a711f7e68cf/src/i18n/i18n.ts#L29) in the admin-UI repo.
```

Comment:
I would suggest using this: https://github.com/opencast/opencast-admin-interface/blob/main/src/i18n/languages.ts#L29

It's not a permalink, so it might rot by us moving the file or moving the relevant info out of that file, but at least it gets updated with newly supported and/or dropped languages.

Your previous link was dead for a reason that should never ever happen again. (Us renaming the default branch.) (Although spoiler alert: We might actually want to change it to `develop` at some point to make it more consistent with the https://github.com/opencast/opencast. 🙈)

In reality we should point to something that is not code at all, but I couldn't really find anything else myself, sadly. :(